lib/src/main/java/uk/gov/di/ipv/cri/address/library/models/ordinancesurvey/Header.java

```diff
@@ -0,0 +1,148 @@
+package uk.gov.di.ipv.cri.address.library.models.ordinancesurvey;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Header {
+
+    /** (Required) */
+    @SerializedName("uri")
+    @Expose
+    private String uri;
+    /** (Required) */
```

Comment:
This is me not knowing are the /** (Required )* / autogenerated, if not what is the significance